includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -1549,7 +1549,7 @@ protected function build_shipping_methods( $shipping_methods ) {
 	 *
 	 * @param boolean $itemized_display_items Indicates whether to show subtotals or itemized views.
 	 */
-	protected function build_display_items( $itemized_display_items = false ) {
+	public function build_display_items( $itemized_display_items = false ) {
```

Comment:
Why the change from `protected` to `public` here?

Reply:
Because I wanted to united test it and have `$itemized_display_items = true`.